common/src/tx/trust.rs

```diff
@@ -48,29 +48,33 @@ impl FromHex for OwnedNamespace {
     Debug, Clone, Default, PartialEq, Serialize, Deserialize, RlpEncodable, RlpDecodable, Getters,
 )]
 #[getset(get = "pub")]
+#[rlp(trailing)]
 pub struct TrustUpdate {
     trust_id: OwnedNamespace,
     entries: Vec<TrustEntry>,
+    seq_number: Option<u64>,
```

Comment:
Is this `seq_number` planned for next PR?
I don't see any use of this field in this PR.

Reply:
Yes, reserved for future updates